chain/chunks/src/shards_manager_actor.rs

```diff
@@ -838,34 +852,11 @@ impl ShardsManagerActor {
         }
     }
 
-    pub fn receipts_recipient_filter<T>(
```

Comment:
This one is not used anywhere. I did not want to create a separate PR just for that.